authors/apps/articles/serializers.py

```diff
@@ -1,10 +1,12 @@
 from rest_framework import serializers
 
-from authors.apps.profiles.serializers import AuthorSerializer
-from authors.apps.profiles.models import UserProfile
 from taggit_serializer.serializers import (
     TagListSerializerField, TaggitSerializer)
 
+from authors.apps.profiles.models import UserProfile
+from authors.apps.profiles.serializers import AuthorSerializer
+from authors.apps.favorites.models import Favorite
+
```

Comment:
remove this blank line. let all your local imports should be together

Reply:
@sharkdevs , thanks for reviewing. I have made the requested changes.